tests/sqlparser_mysql.rs

```diff
@@ -1817,6 +1817,14 @@ fn parse_hex_string_introducer() {
     )
 }
 
+#[test]
+fn parse_call() {
+    mysql().verified_stmt("CALL my_procedure()");
+    mysql().verified_stmt("CALL my_procedure(1, 'a')");
+    mysql().verified_stmt("CALL my_procedure(1, 'a', @my_var)");
+    mysql().verified_stmt("CALL my_procedure");
+}
+
```

Comment:
It doesn't look like the `CALL` keyword is gated to `mysql` dialect at all. Can we either add some tests to generic, or gate it behind the mysql dialect? 

I'd prefer adding it to generic as other dialects such as duckdb also support `CALL`.

Reply:
You are right ! I changed the tests.